app/assets/stylesheets/scaffolds.css.scss

```diff
@@ -0,0 +1,69 @@
+body {
+  background-color: #fff;
+  color: #333;
+  font-family: verdana, arial, helvetica, sans-serif;
+  font-size: 13px;
+  line-height: 18px;
+}
+
+p, ol, ul, td {
+  font-family: verdana, arial, helvetica, sans-serif;
+  font-size: 13px;
+  line-height: 18px;
+}
+
+pre {
+  background-color: #eee;
+  padding: 10px;
```

Comment:
__[scss_lint]__
* [PropertySortOrder] Properties should be ordered background-color, font-size, padding

<!-- Comment by [SideCI](https://sideci.com) -->